canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -898,6 +899,22 @@ export default class PropertiesController {
 		if (!skipValidateInput) {
 			conditionsUtil.validateInput(inPropertyId, this);
 		}
+		// check enum filters and filter out any invalid values
+		const propertyValue = this.getPropertyValue(propertyId);
```

Comment:
This was added to filter out values that can get added by default.  This shows up in tables when we add default rows of values and sometimes those values should be filtered.

Reply:
I see when we click `Add Value`, all the existing rows get updated to the new filtered list. Should the filter only apply to new rows added?

Similarly, if I enable the filter on a table and click to remove a row, the rows are not updated to the filtered values until I click `Add Value`. Following the same pattern when adding, should the rows be updated when removing a row?

Reply:
I was debating on the best way of handling this.  The checkboxset test is a little odd.  Based on the condition we probably should be updating the values in the table and not just the new rows since users wouldn't be able to update existing rows.

I'll fix this so it works for both add and remove.  I think in most cases the filter wouldn't change once the values are added.